test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,15 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  it("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz')
+  });
 
-  test("returns Buzz when passed a multiple of 5", () => {});
+  it("returns Buzz when passed a multiple of 5", () => {
+    expect(fizzBuzz(5)).toBe('Buzz') 
+  });
 
-  test("returns FizzBuzz when passed a multiple 3 and 5", () => {});
+  it("returns FizzBuzz when passed a multiple 3 and 5", () => {});
 
-  test("returns the number when it isn't a multiple of 3 or 5", () => {});
+  it("returns the number when it isn't a multiple of 3 or 5", () => {});
```

Comment:
As above, you are already handling this logic in your function so its a shame not to see it being tested here:
```
  it("returns the number when it isn't a multiple of 3 or 5", () => {
    expect(fizzBuzz(8)).toBe(8);
    expect(fizzBuzz(22)).toBe(22);
  });
```